BREAKING.md

```diff
@@ -0,0 +1,57 @@
+# Planned breaking changes
+
+## Instructions for PR authors
+
+Never merge a breaking change unless the next release bumps the minor version number (e.g. 0.18 => 0.19).
+
+To plan a breaking change, do the following:
+
+1. Add a deprecation warning
```

Comment:
should we link the contribution, that explains how we track the deprecations?

Reply:
added